### PR TITLE
Better RAM management but with picture in picture & multi window support.

### PR DESCRIPTION
**Pull request by wednesday2024**
I removed the line which enables low RAM support. Which after removing it, the device can now have support for the picture in picture and multi window functions. I still left some lines in the code. It will help with RAM management, like described before, without effecting your day to day uses/functions.

### CODE REVIEW

**Comment by Linux4** [comment]
I'd rather not merge this at the current point, as I had already pointed out in your other pull request.
Why do you insist on getting this merged even though gta4xl* isn't low RAM?

File: system.prop [hunk 1]
```diff
@@ -16,3 +16,22 @@ lockscreen.rot_override=true
 ### WIFI
 wifi.direct.interface=p2p0
 ro.boot.wificountrycode=00
+
+# Free up RAM by purging assets.
+persist.sys.purgeable_assets=1
+
+# Better RAM management
+ro.HOME_APP_ADJ=1
```
Comment (Linux4) [inline system.prop]: Also this isn't related to "RAM management"
It's for pinning home app to memory on **older android versions** and as such it even does the opposite of what you wanted to do (if it still worked on Q)

**Comment by wednesday2024** [comment]
> I'd rather not merge this at the current point, as I had already pointed out in your other pull request.
> Why do you insist on getting this merged even though gta4xl* isn't low RAM?

Because I prefer to have the changes in my personal builds. If it's in your repo, it will make my life easier so I don't have to take any updates that you push to my forked version of your repo.

File: system.prop (hunk 1 above)
Comment (wednesday2024) [reply system.prop]: It says # Better RAM management?

Comment (Linux4) [reply system.prop]: But that's not what it actually does as I said.

**Comment by Linux4** [comment]
> > I'd rather not merge this at the current point, as I had already pointed out in your other pull request.
> > Why do you insist on getting this merged even though gta4xl* isn't low RAM?
> 
> Because I prefer to have the changes in my personal builds. If it's in your repo, it will make my life easier so I don't have to take any updates that you push to my forked version of your repo.

You know that you can just merge my repo or even rebase your changes on top of mine again (If you don't like the merge commits)?

**Comment by wednesday2024** [comment]
> > > I'd rather not merge this at the current point, as I had already pointed out in your other pull request.
> > > Why do you insist on getting this merged even though gta4xl* isn't low RAM?
> > 
> > 
> > Because I prefer to have the changes in my personal builds. If it's in your repo, it will make my life easier so I don't have to take any updates that you push to my forked version of your repo.
> 
> You know that you can just merge my repo or even rebase your changes on top of mine again (If you don't like the merge commits)?

I have done that before. It broke the code. I submitted a new patch.

File: system.prop (hunk 1 above)
Comment (wednesday2024) [reply system.prop]: > But that's not what it actually does as I said.

I looked it up, apparently, it is supposed to add more files to your RAM.

**Comment by Linux4** [comment]
Sorry but the new props you added are just a lot of random props (some of these even moved to overlays in newer android versions) and most of these aren't even for battery saving (so your commit message doesn't really match the changes).
Where did you copy all these from?
Sorry but I'm definitely not going to merge this.

**Comment by wednesday2024** [comment]
> Sorry but the new props you added are just a lot of random props (some of these even moved to overlays in newer android versions) and most of these aren't even for battery saving (so your commit message doesn't really match the changes).
> Where did you copy all these from?
> Sorry but I'm definitely not going to merge this.

I used this link: https://www.rankred.com/android-build-prop-tweaks/
and
this link: https://techspreaderofficial.blogspot.com/2020/04/useful-android-build-prop-tweaks-for.html

**Comment by Linux4** [comment]
Sorry, as said before I won't accept props randomly copied from some random websites.
Most of these don't apply to Q anymore or are even OEM specific.

**Pull request closed without merging**